### PR TITLE
[docs] Updates to API reference content

### PR DESCRIPTION
**Pull request by kramantab**
Addressing the following Github issues:

- [Add content and sample for uploading file objects directly to workbook publish endpoint](https://github.com/tableau/server-client-python/issues/768)
- https://github.com/tableau/server-client-python/issues/1007
- https://github.com/tableau/server-client-python/issues/848

### CODE REVIEW

File: docs/api-ref.md [hunk 1]
```diff
@@ -3188,7 +3227,8 @@ Attribute | Description
...
+`user_quota`| (Optional) Specifies the total number of users for the site. The number can't exceed the number of licenses activated for the site; and if tiered capacity attributes are set, then `user_quota` will equal the sum of the tiered capacity values, and attempting to set `user_quota` will cause an error. 
+Tiered capacity attributes: `tier_explorer_capacitytier_explorer_capacity` `tier_creator_capacitytier_creator_capacity` `tier_viewer_capacity`| (Optional) The maximum number of licenses for users with the Creator, Explorer, or Viewer role, respectively, allowed on a site.(Optional) The maximum number of licenses for users with the Creator, Explorer, or Viewer role, respectively, allowed on a site.
```
Comment (jacalata) [inline docs/api-ref.md:3231]: Is this line doubled somehow? It looks odd. Same with hidden_views below.

Comment (kramantab) [reply docs/api-ref.md:3231]: @jacalata - sorry which line are you specifically referring to?  Somehow I don't see the line your comments are attached to.

Comment (kramantab) [reply docs/api-ref.md:3231]: @jacalata - I see the duplication now.  Will fix it.  Thanks!

**Comment by kramantab** [comment]
Fixed the duplicate issue and now merging

File: docs/api-ref.md [hunk 2]
```diff
@@ -3188,7 +3227,8 @@ Attribute | Description
...
-`user_quota`| (Optional) Specifies the maximum number of users for the site. If you do not specify this value, the limit depends on the type of licensing configured for the server. For user-based license, the maximum number of users is set by the license. For core-based licensing, there is no limit to the number of users. If you specify a maximum value, only licensed users are counted and server administrators are excluded. Mutually exclusive with tiered license level settings.
+`user_quota`| (Optional) Specifies the total number of users for the site. The number can't exceed the number of licenses activated for the site; and if tiered capacity attributes are set, then `user_quota` will equal the sum of the tiered capacity values, and attempting to set `user_quota` will cause an error. 
+Tiered capacity attributes: `tier_explorer_capacitytier_explorer_capacity` `tier_creator_capacitytier_creator_capacity` `tier_viewer_capacity`| (Optional) The maximum number of licenses for users with the Creator, Explorer, or Viewer role, respectively, allowed on a site.
```
Comment (jacalata) [inline docs/api-ref.md:3231]: I think you still need to fix these two names: `tier_explorer_capacitytier_explorer_capacity` `tier_creator_capacitytier_creator_capacity

**Review by jacalata — CHANGES_REQUESTED** [review]

File: docs/api-ref.md (hunk 2 above)
Comment (kramantab) [reply docs/api-ref.md]: It should be fixed now.  Thank you!

**Review by jacalata — APPROVED** [review]

**Pull request merged**